service/csv/pokemon.go

```diff
@@ -0,0 +1,225 @@
+package service_csv
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/rayl17/academy-go-q42021/model"
+
+	parameters "github.com/rayl17/academy-go-q42021/global"
+)
+
+type ServiceInterface interface {
+	GetPokemonByName(name string) (*model.Pokemon, error)
+	GetPokemonByID(id string) (*model.Pokemon, error)
+	getPokemons() []model.Pokemon
+	SearchPokemonApi(name string) (string, error)
+	GetPokemonsByTypes(types string, items int, items_per_worker int) (*[]model.Pokemon, error)
+}
+
+type PokemonService struct {
+	servicePath string
+}
+
+func NewPokemonService(path string) (*PokemonService, error) {
+
+	return &PokemonService{
+		servicePath: path,
+	}, nil
+}
```

Comment:
If not returning an error, why declaring it?

```suggestion
func NewPokemonService(path string) (*PokemonService, error) {
	return &PokemonService{
		servicePath: path,
	}, nil
}
```